user-docs/source/instructions.rst

```diff
@@ -0,0 +1,67 @@
+Instructions
+************
+
+In this section we'll take you through the process of contributing data to the ABCD-J catalog.
+In essence, it involves filling metadata properties into sheet-based forms and submitting the
+completed forms.
+
+
+Step 1 - View the example sheets
+================================
+
+To familiarize yourself with the metadata format, we have constructed a complete metadata
+record that is ready for submission to a data catalog. You can access this in one of two ways:
+
+* in an online `example Google Sheets document`_
+* by downloading this :download:`example Excel file <_static/abcdj-example-record.xlsx>`
+
+The example record is about the `Palmer Penguins dataset`_. You will see that the document
+contains several sheets, such as ``dataset``, ``authors``, etc. We will expand on these sections
+below.
+
+.. _example Google Sheets document: https://docs.google.com/spreadsheets/d/1YNZV5_kSa9HS8iB8bfSBQf9_sMr4d3cl
+.. _Palmer Penguins dataset: https://allisonhorst.github.io/palmerpenguins/
+
+Step 2 - Access the template sheets
+===================================
+
+When filling in the metadata properties for your dataset, you can use empty template sheets
+to speed up the process. The template sheets can be accessed in one of two ways:
+
+* via the `template Google Sheets document`_ (download once the tab is open)
+* via the :download:`template Excel file <_static/abcdj-template-record.xlsx>`
+
+.. _template Google Sheets document: https://docs.google.com/spreadsheets/d/1LNeiVilsA-2EEvDjKr1FdibTMx78tTMy
```

Comment:
this sheet links the sfb1451 docs, is that intentional?

Reply:
Those links will be updated once this documentation is live.